frog/imports/ui/GraphEditor/SidePanel/ActivityPanel.js

```diff
@@ -75,6 +76,7 @@ const EditClass = props => {
       >
         <div />
       </Form>
+      <FileForm />
```

Comment:
Right now it hard to make the difference between what part of the sidePanel is for the config and what part is the fileUploaderTool. Can you:
- Stick the tool to the bottom of the SidePanel
- Have a toggle button for the tool